packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -423,6 +425,7 @@ function getFormBodyAndHeaders(
     odspResolvedUrl: IOdspResolvedUrl,
     storageToken: string,
     snapshotOptions: ISnapshotOptions | undefined,
+    otherFormOptions?: {[index: string]: string},
```

Comment:
Not sure if "other" wording is useful here. I'd just leave it at "headers", as that's what it is (even though they are encoded as form options)